commit cd540468a63676ed7c979ac0eb548b6d1324d075
Author: Robert Noland <rnoland@2hip.net>
Date:   Tue Nov 3 11:52:57 2009 -0600

    Order the include directories so that local includes get picked up first.

diff --git a/src/atlantis/Makefile.am b/src/atlantis/Makefile.am
index e217477..ac2fb4f 100644
--- a/src/atlantis/Makefile.am
+++ b/src/atlantis/Makefile.am
@@ -40,12 +40,12 @@ dist_libatlantis_la_SOURCES = \
 BUILT_SOURCES = $(nodist_libatlantis_la_SOURCES)
 
 INCLUDES =                              \
-	@COMPIZ_CFLAGS@                  \
+	-I$(top_srcdir)/include             \
+	@COMPIZ_CFLAGS@                     \
 	-DDATADIR='"$(compdatadir)"'        \
 	-DLIBDIR='"$(libdir)"'              \
 	-DLOCALEDIR="\"@datadir@/locale\""  \
-	-DIMAGEDIR='"$(imagedir)"'          \
-	-I$(top_srcdir)/include
+	-DIMAGEDIR='"$(imagedir)"'
 
 moduledir = $(plugindir)
 
diff --git a/src/cubemodel/Makefile.am b/src/cubemodel/Makefile.am
index c9b1369..866082c 100644
--- a/src/cubemodel/Makefile.am
+++ b/src/cubemodel/Makefile.am
@@ -14,12 +14,12 @@ dist_libcubemodel_la_SOURCES = \
 BUILT_SOURCES = $(nodist_libcubemodel_la_SOURCES)
 
 INCLUDES =                              \
-	@COMPIZ_CFLAGS@                  \
+	-I$(top_srcdir)/include             \
+	@COMPIZ_CFLAGS@                     \
 	-DDATADIR='"$(compdatadir)"'        \
 	-DLIBDIR='"$(libdir)"'              \
 	-DLOCALEDIR="\"@datadir@/locale\""  \
-	-DIMAGEDIR='"$(imagedir)"'          \
-	-I$(top_srcdir)/include
+	-DIMAGEDIR='"$(imagedir)"'
 
 moduledir = $(plugindir)
 
diff --git a/src/elements/Makefile.am b/src/elements/Makefile.am
index cfb6a70..b675fec 100644
--- a/src/elements/Makefile.am
+++ b/src/elements/Makefile.am
@@ -17,12 +17,12 @@ dist_libelements_la_SOURCES = \
 BUILT_SOURCES = $(nodist_libelements_la_SOURCES)
 
 INCLUDES =                              \
-	@COMPIZ_CFLAGS@                  \
+	-I$(top_srcdir)/include             \
+	@COMPIZ_CFLAGS@                     \
 	-DDATADIR='"$(compdatadir)"'        \
 	-DLIBDIR='"$(libdir)"'              \
 	-DLOCALEDIR="\"@datadir@/locale\""  \
-	-DIMAGEDIR='"$(imagedir)"'          \
-	-I$(top_srcdir)/include
+	-DIMAGEDIR='"$(imagedir)"'
 
 moduledir = $(plugindir)
 
diff --git a/src/fakeargb/Makefile.am b/src/fakeargb/Makefile.am
index 0a4f275..8cbe3ed 100644
--- a/src/fakeargb/Makefile.am
+++ b/src/fakeargb/Makefile.am
@@ -9,12 +9,12 @@ dist_libfakeargb_la_SOURCES = fakeargb.c
 BUILT_SOURCES = $(nodist_libfakeargb_la_SOURCES)
 
 INCLUDES =                              \
-	@COMPIZ_CFLAGS@                  \
+	-I$(top_srcdir)/include             \
+	@COMPIZ_CFLAGS@                     \
 	-DDATADIR='"$(compdatadir)"'        \
 	-DLIBDIR='"$(libdir)"'              \
 	-DLOCALEDIR="\"@datadir@/locale\""  \
-	-DIMAGEDIR='"$(imagedir)"'          \
-	-I$(top_srcdir)/include
+	-DIMAGEDIR='"$(imagedir)"'
 
 moduledir = $(plugindir)
 
diff --git a/src/mswitch/Makefile.am b/src/mswitch/Makefile.am
index 55439ba..28f7732 100644
--- a/src/mswitch/Makefile.am
+++ b/src/mswitch/Makefile.am
@@ -9,12 +9,12 @@ dist_libmswitch_la_SOURCES = mswitch.c
 BUILT_SOURCES = $(nodist_libmswitch_la_SOURCES)
 
 INCLUDES =                              \
-	@COMPIZ_CFLAGS@                  \
+	-I$(top_srcdir)/include             \
+	@COMPIZ_CFLAGS@                     \
 	-DDATADIR='"$(compdatadir)"'        \
 	-DLIBDIR='"$(libdir)"'              \
 	-DLOCALEDIR="\"@datadir@/locale\""  \
-	-DIMAGEDIR='"$(imagedir)"'          \
-	-I$(top_srcdir)/include
+	-DIMAGEDIR='"$(imagedir)"'
 
 moduledir = $(plugindir)
 
diff --git a/src/snow/Makefile.am b/src/snow/Makefile.am
index 4440202..3c691f9 100644
--- a/src/snow/Makefile.am
+++ b/src/snow/Makefile.am
@@ -9,12 +9,12 @@ dist_libsnow_la_SOURCES = snow.c
 BUILT_SOURCES = $(nodist_libsnow_la_SOURCES)
 
 INCLUDES =                              \
-	@COMPIZ_CFLAGS@                  \
+	-I$(top_srcdir)/include             \
+	@COMPIZ_CFLAGS@                     \
 	-DDATADIR='"$(compdatadir)"'        \
 	-DLIBDIR='"$(libdir)"'              \
 	-DLOCALEDIR="\"@datadir@/locale\""  \
-	-DIMAGEDIR='"$(imagedir)"'          \
-	-I$(top_srcdir)/include
+	-DIMAGEDIR='"$(imagedir)"'
 
 moduledir = $(plugindir)
 
diff --git a/src/tile/Makefile.am b/src/tile/Makefile.am
index 03b7289..9eec1c1 100644
--- a/src/tile/Makefile.am
+++ b/src/tile/Makefile.am
@@ -9,12 +9,12 @@ dist_libtile_la_SOURCES = tile.c
 BUILT_SOURCES = $(nodist_libtile_la_SOURCES)
 
 INCLUDES =                              \
-	@COMPIZ_CFLAGS@                  \
+	-I$(top_srcdir)/include             \
+	@COMPIZ_CFLAGS@                     \
 	-DDATADIR='"$(compdatadir)"'        \
 	-DLIBDIR='"$(libdir)"'              \
 	-DLOCALEDIR="\"@datadir@/locale\""  \
-	-DIMAGEDIR='"$(imagedir)"'          \
-	-I$(top_srcdir)/include
+	-DIMAGEDIR='"$(imagedir)"'
 
 moduledir = $(plugindir)
 
